src/Engines/QueryBuilderEngine.php

```diff
@@ -340,10 +341,13 @@ protected function compileRelationSearch($query, $relation, $column, $keyword)
                 $relationMethod = "whereRaw";
             }
 
+            //overwrite the first element of the binding - we expect the keyword to be located there
+            $bindings[0]  = $this->prepareKeyword($keyword);
```

Comment:
@yajra I guess you could say that it's a bit hackish, I thought that the 'LIKE' is being build somewhere else but looking at `compileQuerySearch` I see that `prepareKeyword` is already being called on the keyword we don't need to re-add it - maybe we can skip this hackish line.

I didn't do this since I don't have this type of relationships to test but for `MorphToMany` it would be required to merge the `$relationType->getMorphClass()` and `$bindings` arrays together in the right order - but this would need some testing.